config/default.yaml

```diff
@@ -1,6 +1,7 @@
 base_url: http://localhost:3000
 terrain_url: http://terrain
 listen_port: 3000
+user_portal_url: https://user.cyverse.org
```

Comment:
I noticed we have a different URL for the portal API further down in this file. I don't know if we should have both things, but if we do the defaults should probably match (and that one uses `portaldev`, looks like)

Reply:
Ah, yes, the defaults should match. Good catch. As for whether or not we should have both, that's another question. I guess I was being lazy and I thought I'd ensure that they were the same when generating the configuration file. On second thought, however, I think it does make sense to have just one configuration setting for both. I'll fix that.